spec/controllers/medicaid/intro_citizen_controller_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
+
+RSpec.describe Medicaid::IntroCitizenController, type: :controller do
+  describe "#next_path" do
```

Comment:
may want to look at the specs for `IntroCollegeController` -- we make sure that the member for a single member household is updated via this controller, which I think we should be doing for all single member households on these "yes/no" pages.